website/src/pages/flu/[...slug].astro

```diff
@@ -0,0 +1,9 @@
+---
+import { ServerSide } from '../../views/serverSideRouting';
+
+// This serves as a catch-all route for the /flu/* path
+return Astro.redirect(
+    `${ServerSide.routing.getOrganismView('influenza-a.sequencingEffortsView').pageStateHandler.getDefaultPageUrl()}`,
```

Comment:
This might be a bit surprising. We could also make it this:
```suggestion
    `/influenza-a/${Astro.params.slug}`,
```

Reply:
Or maybe we could simply redirect to the new index pages that #621 adds?

(In general, #621 and this PR will have conflicts, depending on which one will be merged first)

Reply:
I actually quite like your suggestion - but Im fine with waiting till the other PR is merged and then resolving merge conflicts after

Reply:
The other PR is merged - could you please rebase again?